### DISCUSSION TITLE
Where to find Arrow Flight interface documentation?

### DISCUSSION THREAD

**Discussion by kesavkolla** [unum-cloud/UStore#254] [Q&A]
I want to use graph and key/value modalities from my application. My application is written in rust and I would like to use arrow flight as there is no native rust client yet. Are there any examples of arrow flight that I can refer to?

**Comment by ashvardanian** [comment]
We don't have anything Rust-related yet, but a quick Google search brings [some Crates](https://crates.io/crates/arrow-flight). Is that what you were searching for?

**Comment by kesavkolla** [comment]
Yes I can get arrow flight client libraries in rust. I am wondering is there any doc on how to use different modalities with flight?

**Comment by ashvardanian** [comment]
Not at this point, unfortunately. We have examples showing how to [work with binary data in Python](https://github.com/unum-cloud/ukv/blob/main/docs/flight/ddos.py). We can extend those examples with @davvard to cover Graphs as well.
